auto_submit/lib/service/revert_issue_body_formatter.dart

```diff
@@ -39,6 +41,9 @@ class RevertIssueBodyFormatter {
     revertPrBody = '''
 $revertPrLink
 Initiated by: $initiatingAuthor
+
+Reason: $revertReason
```

Comment:
Is it easy to add the initial author here in the body? Context: https://github.com/flutter/flutter/issues/137156
If not, feel free to skip here.

Reply:
+1 to that. Mentioning authors and reviewers here would be awesome! (not a blocker for this PR, though)

Reply:
Yes, that should not be hard to do.

Reply:
@goderbauer for authors, I assume this is anyone who landed a commit to the pull request? And for reviews, are we talking just the approvals?

Reply:
For authors, it would be enough to just mention the person who created the original PR. If it contains contributions from others, I don't think those need to be mentioned.
For reviewers, only those that have approved the PR would be good.

Reply:
You got it! Thank you for clarifying!